### ISSUE TITLE
no usable prompt/input area when restarting kernel

### ISSUE THREAD

**Issue by vv111y** [dzop/emacs-jupyter#50]
When restarting the kernel, sometimes I am unable to enter any text.

![screen shot 2019-03-01 at 9 30 46 pm](https://user-images.githubusercontent.com/5576943/53675887-67125c80-3c69-11e9-821c-63f0f5113da3.png)

I get errors like this:
```
Error running timer ‘jupyter-handle-message’: (error "Found end of previous cell")
jupyter-repl-cell-beginning-position: Found end of previous cell [2 times]
funcall-interactively: Text is read-only
Company: An error occurred in auto-begin
Company: backend company-capf error "Found end of previous cell" with args (prefix)
Mark set
jupyter-repl-cell-beginning-position: Found end of previous cell
helm-M-x: Found end of previous cell
```

**Comment by dzop** [comment]
Yes, it looks like restarting is still an issue. Also see #28.

**Comment by dzop** [comment]
Closing since I believe issues with restarting the REPL have been fixed in 1f5c7743d534929efaad7592762f241700bc1f6c.
